extensions/amp-story/0.1/amp-story-bookend.css

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import './amp-story-share.css';
+
+.i-amphtml-story-bookend {
+  text-align: left !important;
+  color: #fff !important;
+  bottom: 0 !important;
+  display: flex !important;
+  flex-direction: column !important;
+  left: 0 !important;
+  position: absolute !important;
+  right: 0 !important;
+  top: 0 !important;
+  z-index: 100001 !important;
+  transition: transform 0.3s ease-out !important;
+}
+
+.i-amphtml-story-bookend.i-amphtml-hidden {
+  transition-timing-function: ease-in !important;
+  transform: translateY(100vh) !important;
+  pointer-events: none !important;
+}
+
+.i-amphtml-story-bookend-hidden {
+  pointer-events: none !important;
+}
+
+.i-amphtml-story-bookend-overflow {
+  overflow: auto !important;
+  margin-top: auto !important; /* positions it at the bottom of container */
+  transition: opacity 0.3s, transform 0.3s ease-out !important;
+}
+
+.i-amphtml-story-bookend-hidden .i-amphtml-story-bookend-overflow {
+  transform: translateY(200px) !important;
+  opacity: 0 !important;
+}
+
+.i-amphtml-story-bookend-inner {
+  margin: 88px 0 0 !important;
+  position: relative !important;
+  padding-top: 32px !important;
+  overflow: hidden !important;
+}
+
+.i-amphtml-story-bookend-inner::before {
+  content: " " !important;
+  display: block !important;
+  z-index: -1 !important; /* places it under content */
+  background: rgba(0, 0, 0, 0.75) !important;
+  position: absolute !important;
+  left: 8px !important;
+  top: 8px !important;
+  bottom: 8px !important;
+  right: 8px !important;
+  border-radius: 8px !important;
+}
+
+.i-amphtml-story-bookend-fullbleed .i-amphtml-story-bookend-inner::before,
+[desktop] .i-amphtml-story-bookend-inner::before {
```

Comment:
I thought `[desktop]` (and other selectors relying on the host context) didn't work?

Reply:
Nevermind, I see where you duplicated it inside the shadow root 👍